lib/ExpensiMark.js

```diff
@@ -628,4 +612,25 @@ export default class ExpensiMark {
         }
         return replacedText;
     }
+
+    /**
+     * Modify text for Quote content and translate it into blockquote html if it matches the regex.
```

Comment:
Can we make this doc and method name more descriptive? Maybe call it `formatWhitespaceForQuote`?

Reply:
Updated